atlas/src/components/ErrorFallback.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { FallbackProps } from 'react-error-boundary'
+import { Button } from '@/shared/components'
+import { sizes, colors } from '@/shared/theme'
+
+const Container = styled.div`
+  padding: ${sizes.b4};
+  color: ${colors.gray[400]};
+  display: grid;
+  place-items: center;
+`
+
+const StyledButton = styled(Button)`
+  color: ${colors.white};
+`
+
+const ErrorFallback: React.FC<FallbackProps> = ({ error, resetErrorBoundary }) => {
+  return (
```

Comment:
There is a screen in Figma showing how the "error page" should look like: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=2155%3A180756

Also, we probably need a distinction between a full view error (for example if the video fails to load in video view) and a component-level error as the one we'd display instead of a carousel

Reply:
I had totally missed that, thanks. I've changed the error fallback for a route to that and kept the previous ones for galleries.